lib/ae_mdw/txs.ex

```diff
@@ -61,6 +61,23 @@ defmodule AeMdw.Txs do
     end
   end
 
+  def count(state, nil, %{"id" => id}) do
+    case Validate.id(id) do
+      {:ok, address} ->
+        total_count =
+          Node.tx_types()
+          |> Enum.map(fn tx_type ->
+            field_count(state, tx_type, address)
+          end)
+          |> Enum.sum()
+
+        {:ok, total_count}
+
+      {:error, reason} ->
+        {:error, reason}
+    end
```

Comment:
I have read some code having a `with` without the else and since then I have copied the idea.